lambdas/handlers/logout_handler.py

```diff
@@ -39,16 +40,21 @@ def logout_handler(token):
         remove_session_from_dynamo_db(session_id)
 
     except ClientError as e:
-        logger.error(f"Error logging out user: {e}", {"Result": "Unsuccessful logout"})
+        logger.error(
+            f"{LambdaError.LogoutClient.to_str()}: {e}",
+            {"Result": "Unsuccessful logout"},
+        )
         return ApiGatewayResponse(
-            500, "Error logging user out", "GET"
+            500,
+            LambdaError.LogoutClient.create_error_body(),
+            "GET",
         ).create_api_gateway_response()
     except (jwt.PyJWTError, KeyError) as e:
         logger.error(
-            f"error while decoding JWT: {e}", {"Result": "Unsuccessful logout"}
+            f"{LambdaError.LogoutAuth.to_str()}: {e}", {"Result": "Unsuccessful logout"}
```

Comment:
We've replaced "error while decoding jwt" with ""Invalid Authorization header" is the the correct error for this case?